src/test/java/org/codehaus/mojo/jaxws/MixedITCase.java

```diff
@@ -151,6 +153,7 @@ public void jaxwscommons81_wsgen()
 
     private String getExpectedToolsVersion()
     {
+        /*
```

Comment:
Is this code commented out on purpose and not removed?

Reply:
No, there was no particulary reason...